lib/constants.dart

```diff
@@ -14,3 +14,46 @@ const String aniRight = 'assets/animations/ic_anim_right.gif';
 const String effFlash = 'assets/effects/ic_effect_flash.gif';
 const String effInvert = 'assets/effects/ic_effect_invert.gif';
 const String effMarque = 'assets/effects/ic_effect_marquee.gif';
+
+//list containing the location of all the assets
+List<String> vectors = [
```

Comment:
Instead of maintaining the list of vectors. Can you try to read all vectors from this location? We don't want to maintain a list as well as the asset where we could infer by reading the AssetManifest.json